docs/index.md

```diff
@@ -91,6 +91,32 @@ If a user elects to delete the offline map, the app deletes the offline mobile m
 
 > A user can resume work online without deleting the offline map.
 
+### View and navigate to map bookmarks
+
+Web maps can include a list of bookmarks. Each bookmark consists of a map extent (visible area) and a name. Bookmarks can be authored in ArcGIS Pro and the ArcGIS Web Map Viewer.
+
+You can click the Bookmark icon to see a list of bookmarks in the map. Selecting a bookmark will navigate to the bookmark's extent.
```

Comment:
Because "navigate" is an existing verb in the ArcGIS ecosystem, can we go with something else? Perhaps "set the map view's visible area to the bookmark's extent"?